src/main/scala/org/deepdive/ddlog/DeepDiveLogDeltaDeriver.scala

```diff
@@ -31,46 +31,51 @@ object DeepDiveLogDeltaDeriver{
       )
     }
 
-    var incCqBodies = new ListBuffer[List[Atom]]()
-    var incCqConditions = new ListBuffer[Option[Cond]]()
+    var incCqBodies = new ListBuffer[List[Body]]()
     // New incremental bodies
-    cq.bodies zip cq.conditions foreach { case (body, cond) =>
+    cq.bodies foreach { bodies =>
+      // We don't support deriving delta rules for modifiers
+      bodies foreach {
+        case _: Cond | _: Atom => // supported
+        case _: QuantifiedBody => throw new RuntimeException("Deriving delta rules for modifier atom is not supported!")
```

Comment:
I think `: QuantifiedBody` should be removed and make it a pure wildcard.  The whole point of this check is to reject all other cases than Cond and Atom.